sale_order_color_code_tintometric_system/views/account_move_views.xml

```diff
@@ -0,0 +1,12 @@
+<odoo>
+    <record id="account_move_form_view" model="ir.ui.view">
+        <field name="name">account.move.view.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//tree//field[@name='name']" position="after">
```

Comment:
Use a more specific xpath if possible (pointing to move lines field if possible)